src/api_key.rs

```diff
@@ -3,40 +3,37 @@ use std::str::FromStr;
 
 use base64::Engine;
 use rand::rngs::OsRng;
-use rand::RngCore;
+use rand::Rng;
 use serde::Serialize;
 use sha3::{Digest, Sha3_256};
 
+const SECRET_LEN: usize = 16;
+const UUID_LEN: usize = 16;
+
 #[derive(Debug, Clone, PartialEq, Eq)]
 pub struct ApiKey {
     pub relayer_id: String,
-    pub api_key: [u8; 32],
+    pub secret: [u8; SECRET_LEN],
```

Comment:
We actually need to make it into a `Vec<u8>` or some sort of small vec. We already have 32 byte secret keys running in the dev env

Reply:
Sorry could you elaborate? You mean you'd like to keep backwards compatibility with 32byte secrets? (48byte total for the api key?)

Reply:
Yes, exactly. I see that you already implemented that